src/components/NewTimeEntryInput.vue

```diff
@@ -6,150 +6,302 @@
-            (timeEntry: TimeEntry) => ({
-              label: timeEntry.description,
-              value: timeEntry,
-            })
-          );
-
-          callback(newData);
-        },
-        render(item: TimeEntryAutocompleteItem): HTMLDivElement | undefined {
-          const wrapper = document.createElement("div");
-          const svg = document.createElementNS(
-            "http://www.w3.org/2000/svg",
-            "svg"
-          );
-          const circle = document.createElementNS(
-            "http://www.w3.org/2000/svg",
-            "circle"
-          );
-          const timeEntrySpan = document.createElement("span");
-          const projectWrapper = document.createElement("div");
-          const projectSpan = document.createElement("span");
-
-          const projectName =
-            item.value?.project?.data.name || t("projectIndicator.noProject");
-          const color = item.value?.project?.data.color_hex || "#676767";
-
-          const timeEntryName = item.value.description;
-
-          [
-            "flex",
-            "justify-between",
-            "select-none",
-            "text-gray-900",
-            "text-sm",
-          ].forEach((eleClass) => wrapper.classList.add(eleClass));
-
-          [
-            "flex",
-            "items-center",
-            "text-xs",
-            "select-none",
-          ].forEach((eleClass) => projectWrapper.classList.add(eleClass));
-          projectWrapper.style.color = color;
-
-          svg.setAttribute("width", "10");
-          svg.setAttribute("height", "10");
-          svg.setAttribute("viewBox", "0 0 10 10");
-          svg.setAttribute("fill", "none");
-          svg.setAttribute("xmlns", "http://www.w3.org/2000/svg");
-          svg.classList.add("mr-2");
-
-          circle.setAttribute("cx", "5");
-          circle.setAttribute("cy", "5");
-          circle.setAttribute("r", "5");
-          circle.classList.add("fill-current");
-
-          timeEntrySpan.textContent = timeEntryName;
-          timeEntrySpan.classList.add("block");
-
-          projectSpan.classList.add("block");
-          projectSpan.textContent = projectName;
-
-          svg.appendChild(circle);
-          projectWrapper.appendChild(svg);
-          projectWrapper.appendChild(projectSpan);
-
-          wrapper.appendChild(timeEntrySpan);
-          wrapper.appendChild(projectWrapper);
-
-          return wrapper;
+          let loadedData;
+          if (!text.endsWith("#") && !text.endsWith("@")) {
+            const {
+              data: { data: customers },
+            } = await new TimeEntriesService().include("project").suggest(text);
+            const newData: TimeEntryAutocompleteItem[] = customers.map(
+              (timeEntry: TimeEntry) => ({
+                label: timeEntry.description,
+                value: timeEntry,
+              })
+            );
+
+            loadedData = newData;
+          } else if (text.endsWith("#")) {
```

Comment:
Wenn wir checken ob der input auf `#` endet, ist keine Suche nach Projekten möglich. Beispielsweise wenn ich in den Input `#Müller` eingebe, wird eine Suche nach Zeiteinträgen ausgelöst (da `text.endsWith("#") === false`), anstatt nach Projekten die mit "Müller" beginnen zu suchen. Ich denke eine gute Lösung wäre, zu prüfen, ob in `text` ein `#` oder ein `@` irgendwo vorkommt und falls beides vorkommt jeweils das zu nehmen welches weiter hinten steht. Also z. B. bei Eingabe `abc` wird nach Zeiteinträgen gesucht, bei `abc #` werden Projekte vorgeschlagen, bei `abc #projekt @` werden Kunden vorgeschlagen.